membership.md

```diff
@@ -0,0 +1,170 @@
+# Community membership
```

Comment:
Maybe replace the content of this file with a "this content was moved directly on the Kubeflow website at <link>" when the move will have happened.

Mentioning to avoid 404 of the Octocat, if someone reaches this location (after the move).